lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -633,6 +636,46 @@ def _deploy_environment(self, environment: Environment, log: Logger) -> None:
             except Exception as identifier:
                 self._delete_environment(environment, log)
                 raise identifier
+            if self._azure_runbook.collect_core_stats:
+                self._collect_core_stats(
+                    environment, EnvironmentStatus.Deployed, log, deployment_parameters
+                )
+
+    def _collect_core_stats(
```

Comment:
All information should be in Environment messages already, it doesn't need to create another message.

Reply:
Done